### PR TITLE
test: add extra test for stacker

### PR DESCRIPTION
**Pull request by philipdesmedt**

### CODE REVIEW

**Comment by nieldeckx** [comment]
@philipdesmedt great catch - extra tests have been paying off :)

File: clarity/contracts/arkadiko-freddie-v1-1.clar [hunk 1]
```diff
@@ -623,6 +623,12 @@
 
     (try! (pay-stability-fee vault-id coll-type))
     (burn-partial-debt vault-id (min-of debt (get debt vault)) reserve ft coll-type)
+    ;; (let ((fee (unwrap-panic (pay-stability-fee vault-id coll-type))))
```
Comment (nieldeckx) [inline clarity/contracts/arkadiko-freddie-v1-1.clar:626]: Can you explain why this was removed?

**Review by nieldeckx — COMMENTED** [review]
Great :)

File: clarity/contracts/arkadiko-freddie-v1-1.clar (hunk 1 above)
Comment (philipdesmedt) [reply clarity/contracts/arkadiko-freddie-v1-1.clar:626]: Yes sorry, this was actually added by me, but it wasn't useful in the end I think.. gonna remove this completely & merge

**Pull request merged**